### ISSUE TITLE
Use Buildozer

### ISSUE THREAD

**Issue by FeralBytes** [kivy/kivy-remote-shell#4]
I recommend you provide instructions for using Buildozer instead of distribute.sh, it is much easier.

**Comment by kkwaiser** [comment]
@FeralBytes Could you summarize those steps? I having problems with the distribute.sh install method at the moment.

**Comment by FeralBytes** [comment]
This was a long time ago; but basically if I remember correctly; I used git to clone the kivy-remote-shell repository and then used PIP to install Buildozer. Then I used Buildozer to build and install kivy-remote-shell to my phone via ADB/USB.

**Comment by FeralBytes** [comment]
@akshayaurora Thanks for all of the good work.
